vocab-importer/src/main.py

```
@@ -0,0 +1,226 @@
+        )
+
+    @app.get("/health")
+    async def health_check() -> Dict[str, str]:
+        return {"status": "healthy"}
+
+    @app.post("/import", status_code=status.HTTP_201_CREATED)
+    async def import_vocabulary(
+        file: UploadFile = File(
+            default=None,
+            description="Vocabulary file to import (.json, .txt, .csv, .pdf)"
+        )
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)